adobe/aio-cli-plugin-runtime
# Feat. Support for --copy flag on create action.

## Description



## Related Issue
https://github.com/adobe/aio-cli-plugin-runtime/issues/134






## Motivation and Context



## How Has This Been Tested?
Given `aio` context: 
1. Create a new action file. 
2. run `aio runtime action create actionName index.js`
3. run `aio runtime action create newActionName --copy actionName`

Expected: 
- `aio runtime action list `should list the new created action
- `aio runtime action invoke newActionName --result` output should equal the output of: `aio runtime action invoke actionName --result`
- 





## Screenshots (if appropriate):

## Types of changes




## Checklist:
